images/fluent-bit/config/template.apko.yaml

```diff
@@ -10,7 +10,7 @@ accounts:
   users:
     - username: nonroot
       uid: 65532
-  run-as: 65532
+  run-as: 0
```

Comment:
nit: maybe add your findings in the excellent PR description to an issue, then point to it from here?

Reply:
done!

Reply:
Hmm, the issue is private and this is in a public repo 😞

Reply:
Can we include a `tl;dr` inline maybe?

Reply:
Ah good catch 😓 I ended up linking to this PR description